test/tests/restart/kernel_restartable/tests

```diff
@@ -10,6 +10,7 @@
       input = 'kernel_restartable.i'
       exodiff = 'kernel_restartable_out.e'
       detail = "perform residual calculations that accumulate state and"
+      recover = false
```

Comment:
Why?

Reply:
Restart is already tested below in `[test2]`.